### DISCUSSION TITLE
Still asking me to signup after a long time of usage

### DISCUSSION THREAD

**Discussion by yahu1031** [community/community#15947] [Copilot Conversations]
I don't know what happened, but suddenly my GitHub copilot account asked me to signup. But when I try doing it, It says I already have access.

This is from VSCode

![image](https://user-images.githubusercontent.com/35523357/166395656-3f72feef-c216-4774-8a17-a12bc4b0c715.png)

This is from browser

![image](https://user-images.githubusercontent.com/35523357/166395673-77716b81-2add-4348-a6dd-293c8152991a.png)

**Comment by davecheney** [comment]
@yahu1031 your copilot invite is active. Please try the following

1. <img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
   Please try signing out, copilot will prompt you to sign back in.
3. If that doesn't work, please update your extension and double check that you are using VSCode, not VSCodium.
